### DISCUSSION TITLE
thanks for pydantic

### DISCUSSION THREAD

**Discussion by denisart** [pydantic/pydantic#4822] [Projects using Pydantic]
Hello, thanks for pydantic. I use it often in my projects.

I created my project for pydantic model generation from some graphql schema. And for graphql queries generation from python classes. The project URL is https://github.com/denisart/graphql2python

My project so far has few settings. This will be fixed in future releases. Now you can generate a file with pydantic classes from some GraphQL schema.
You can see examples in
data-model generation: https://github.com/denisart/graphql2python/blob/master/docs/source/model.rst
GraphQL queries generation: https://github.com/denisart/graphql2python/blob/master/docs/source/query.rst

This is a thank you post for your project.

**Comment by samuelcolvin** [comment]
Looks great, congratulations.

Thanks so much for letting us know.
